wrappers/core18.go

```diff
@@ -657,6 +661,35 @@ func undoSnapdDbusActivationOnCore() error {
 	return err
 }
 
+var snapdDesktopFileNames = []string{
```

Comment:
as discussed with @jhenstridge, maybe these lists should have a comment about them needing to be append only, as to support removing unused files on update ?

Reply:
Added comment.